data-plane/config/channel/500-dispatcher.yaml

```diff
@@ -128,9 +128,6 @@ spec:
               cpu: 1000m
               # 600Mi for virtual replicas + 100Mi overhead
               memory: 700Mi
-            limits:
-              cpu: 2000m
-              memory: 1000Mi
```

Comment:
We need to also patch the Java config https://github.com/openshift-knative/eventing-kafka-broker/blob/main/openshift/patches/remove_broker_dispatcher_limits.patch